src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -48,7 +48,7 @@ public function __construct(Connection $conn)
     /**
      * {@inheritdoc}
      */
-    public function loadTokenBySeries($series)
+    public function loadTokenBySeries(string $series)
```

Comment:
💡 These changes to `doctrine-bridge` 5 break compatibility with `security-core` 4.4, thus the bump in composer.json. Can be reverted and re-applied later, if you don't want to bump yet.